package.json

```diff
@@ -6,7 +6,7 @@
     "action:help": "npm run action list",
     "action:list": "npm run action list",
     "action": "node ./src/build/run_action.mjs",
-    "clean": "rimraf output node_modules www platforms plugins third_party/jsign/*.jar",
+    "clean": "rimraf build output node_modules www platforms plugins third_party/jsign/*.jar",
```

Comment:
Why do we have `build` and `output`?
I think it's clearer if we have a single build output directory